samples/source_adapter_with_json_metadata/metric.py

```diff
@@ -0,0 +1,33 @@
+"""Analytics module."""
+from savant.gstreamer import Gst
+from savant.deepstream.meta.frame import NvDsFrameMeta
+from savant.deepstream.pyfunc import NvDsPyFuncPlugin
+
+
+class IOU(NvDsPyFuncPlugin):
+    """IOU metric for object detection."""
+
+    def __init__(self, ground_truth: str, element_name: str, **kwargs):
+        super().__init__(**kwargs)
+        self.element_name = element_name
+        self.ground_truth = ground_truth
```

Comment:
On the one hand you are right, it will work without writing __init__ explicitly, but then it is less clear how the parameters in the configuration file and process_frame are related. @bwsw what do you think about it? Should we remove __init__ since we don't really need it here?

Reply:
@bwsw doesn't matter from my POV.

Reply:
removed